### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by ocefpaf**
Follow up on https://github.com/pangeo-data/xESMF/pull/248

### CODE REVIEW

File: .github/workflows/pypi.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ on:
   pull_request:
   push:
     branches:
-      - main
+      - master
```
Comment (ocefpaf) [inline .github/workflows/pypi.yaml:7]: I recommend renaming the master branch to main though. But for now this will do.

**Review by aulemahal — APPROVED** [review]

File: pyproject.toml [hunk 2]
```diff
@@ -48,6 +48,7 @@ readme = {file = "README.rst", content-type = "text/x-rst"}
 [tool.setuptools_scm]
 write_to = "xesmf/_version.py"
 write_to_template = "__version__ = '{version}'"
+tag_regex = "^(?P<prefix>v)?(?P<version>[^\\+]+)(?P<suffix>.*)?$"
```
Comment (ocefpaf) [inline pyproject.toml:51]: This is the most common template used by many projects. When releasing we need to create a tag with the `v` like `v2.0`. If that is not desirable I can change it but the `v` helps find the tags b/c they all will start with the same letter.

Comment (huard) [reply pyproject.toml:51]: All good.

File: .github/workflows/pypi.yaml (hunk 1 above)
Comment (huard) [reply .github/workflows/pypi.yaml:7]: Agreed, let's do this for the next release.

**Pull request merged**